test/models/begin_call_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class BeginCallTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+  
+  test 'begin call is post method' do
+    call = BeginCall.new
+    assert('POST', call.method())
```

Comment:
[For DSL methods, like assert, you should omit the parenthesis for the method call](https://github.com/Invoca/style-guide/tree/master/ruby#omit-parentheses-around-parameters-for-methods-that-are-part-of-an-internal-dsl)

You can omit the parenthesis after `method` since there are no parameters.

Use `assert_equal`, not `assert`. `assert` will test the "truthiness" of a value, it does not compare the two values. The second parameter `call.method` would be used as the message for the test failure. This test can never fail though, because in Ruby strings are always "truthy". [Here is a slightly lengthy but good article on conditionals and truthiness in Ruby.](http://ruby-for-beginners.rubymonstas.org/conditionals/nothing_and_truth.html)

Given that this test can never fail, it tells me that it was not run through regular TDD practices. Before adding the logic to your methods you should see the test fail. This usually is done by adding a blank method and then running the test. If you have already written the method before writing the test, then just comment out the body of the method and run the test to make sure you have a failing test beforehand, otherwise the tests don't provide any value.